src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,121 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.ConnectionUtil;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.DataProcessingException;
+import mate.jdbc.model.Manufacturer;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
```

Comment:
```suggestion
        String insertManufacturerRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
```

let's not use abbreviations